app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,395 @@
+* **Prospective Partners:** These developers are currently undergoing an evaluation process. Since these potential partners have not completed the evaluation process, you grant them restricted view-only access. This allows them to review the API specs and gain an understanding of how your system operates. 
+
+### Create an API product
+
+In this scenario, before you can configure developer teams, you must have an API product created. This API product will be for your Pizza Ordering API. This API is used at your pizza company's website to take orders. Delivery companies can hook into this to see when orders are placed so they can have drivers ready. 
+
+{% navtabs %}
+{% navtab Konnect UI %}
+In {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.com/us/runtime-manager), select a runtime group and follow these steps:
+
+1. Select **Gateway Services** from the side navigation bar, then **New Gateway Service**.
+1. From the **Add a Gateway Service** dialog, enter the following to create a new service:
+    * **Name:** `pizza_ordering`
+    * **Upstream URL:** `http://mockbin.org`
+    * Use the defaults for the remaining fields.
+1. Click **Save**. 
+1. To create an API product for your service, navigate to **API Products** in the sidebar, click **Add API Product** and enter `Pizza Ordering` in the **Product Name** field.
+1. Click **Publish to portal** from the **Actions** dropdown menu. This will allow developers to access the API product once you assign them to a team.
+1. Click **New Version** from the **Actions** dropdown menu and enter `v1` in the **Version Name** field and then click **Create**.
+1. Click the **v1** version and then click **Link** to add a service.
+1. Select a runtime from the **Select Runtime Group** dropdown menu, select `pizza_ordering` from the **Gateway Service** dropdown menu, and then click **Save**.
+1. Click the **Edit** icon next to the status of your product version and select **Published** from the **Edit version status** dialog and then click **Save**.
+1. You need to enable app registration if you want the developer in the Authorized Delivery Partners group to be able to consume your APIs by registering apps. You can use an authentication method of your choice in the [enable or disable application registration](/konnect/dev-portal/applications/enable-app-reg/) documentation.
+
+You now have a published API product for your Pizza Ordering API in your Dev Portal.
+
+{% endnavtab %}
+{% navtab API %} 
+{:.note}
+The {{site.konnect_short_name}} API uses [Personal Access Token (PAT)](/konnect/api/#authentication) authentication. You can obtain your PAT from the [personal access token page](https://cloud.konghq.com/global/account/tokens). The PAT must be passed in the `Authorization` header of all requests.
+
+1. Create the `pizza_ordering` service:
+```bash
```

Comment:
@Guaris For when you review this, I feel like some of the formatting in the preview page is still weird even though everything looks correct and is correctly formatted in the markdown. I also tested all of the examples from the preview page, so they all work, the format is just...odd.

Reply:
i pushed an example of this. You have to indent the specific parts for it to work correctly.